app/controllers/api/v1/admin_controller.rb

```diff
@@ -20,8 +20,9 @@ def add_publisher_to_project
 
   # GET /api/admin/user/slack?uid=:uid
   def slack_user
-    user = User.where(provider: 'slack', uuid: params[:uid].to_s).last
-    user = { token: user.token } unless user.nil?
+    user = User.find_with_omniauth(params[:uid].to_s, 'slack')
+    slack_account = user.accounts.where(provider: 'slack').first unless user.nil?
```

Comment:
this will not work in case of multiple Slack logins

Reply:
+1